lib/alerts/components/AlertEditor.js

```diff
@@ -143,91 +136,104 @@ export default class AlertEditor extends Component<Props> {
-              <Row>
-                <Col xs={12} style={{marginTop: '10px'}}>
-                  <FormGroup controlId='formControlsTitle'>
-                    <ControlLabel>
-                      Alert Title
-                      {' '}
+                  onClick={this._onClickBack}>
+                  <Icon type='chevron-left' /> Back
+                </Button>
+              </Col>
+              <Col xs={8} sm={5} md={4}>
+                <ButtonToolbar className='pull-right' style={{marginLeft: '5px'}}>
+                  <Button
+                    title={editButtonMessage}
+                    bsStyle='primary'
+                    disabled={editingIsDisabled}
+                    onClick={this.validateAndSave}
+                  ><Icon type='save' /> Save</Button>
+                  <Button
+                    title={deleteButtonMessage}
+                    bsStyle='danger'
+                    disabled={deleteIsDisabled}
+                    onClick={this._onClickDelete}
+                  ><Icon type='trash' /> Delete</Button>
+                </ButtonToolbar>
+                <FormGroup
+                  className='pull-right'
+                  style={{position: 'relative', top: '5px'}}>
+                  <Toggle
+                    id='alert-published'
+                    disabled={!canPublish}
+                    checked={alert.published}
+                    onChange={this._onClickPublish} />
+                  <label
+                    htmlFor='alert-published'
+                    style={{position: 'relative', top: '-5px', marginLeft: '5px'}}>
+                    Published?
+                  </label>
+                </FormGroup>
+              </Col>
+            </Row>
+  
+            <Row>
+              <Col xs={12} sm={6}>
+                <Row>
+                  <Col xs={12} style={{marginTop: '10px'}}>
+                    <FormGroup controlId='formControlsTitle'>
+                      <ControlLabel>
+                        Alert Title
+                        {' '}
                       <span
                         className={
                           titleCharactersRemaining > CHAR_WARNING_LIMIT
                             ? 'text-muted'
                             : 'text-danger'
                         }
                         style={{fontWeight: 400}}>
-                        {titleCharactersRemaining}
+                        {titleCharacterCount}
                       </span>
                       <h5 style={{margin: '0px'}}>
                         <small>
+                          {titleCharacterCount > ALERT_TITLE_CHAR_LIMIT 
+                            ? (
+                              <span className='text-danger'>
+                                {`WARNING:  Alert title longer than ${ALERT_TITLE_CHAR_LIMIT} characters may get truncated in some dissemination channels. `}
+                              </span>
+                          ) : ''} 
```

Comment:
I will let @landonreed weigh on the final appearance, but could you see whether you could use some of the built-in functionality to display validation errors? See https://react-bootstrap-v3.netlify.app/components/forms/#forms-validation for the version of Bootstrap used for this repo.

Reply:
If you do decide to use the layout as written, I would at least place the error message on a separate line. It is easy to miss otherwise given the low color contrast between dark red and gray.